www/src/pages/components/statusalert.mdx

```diff
@@ -1,13 +1,11 @@
 ---
 title: 'StatusAlert'
 type: 'component'
-status: 'Needs Work'
+status: 'Deprecate Soon'
```

Comment:
Feel like we have so many variations on this alert floating around in the MFEs. :)